posthog/models/user.py

```diff
@@ -94,6 +94,7 @@ def events_column_config_default() -> Dict[str, Any]:
     return {"active": "DEFAULT"}
 
 
+@reversion.register(exclude=("events_column_config", "email"))
```

Comment:
Why email?

Reply:
That (and api token on team) I was thinking to limit the amount of PII that this could leak. 

May be premature...?

And would mean we couldn't audit when email or api token changed 🤔

Reply:
Actually, let's tread carefully and modify as few models in the first PR as possible. I suggest just `insight` and `feature_flag`. We can add more later, but there are too many "now I must think about this" moments per PR here 😝.